vertex_lcfi/util/inc/memorymanager.h

```diff
@@ -72,9 +72,9 @@ namespace vertex_lcfi
 	//Protect the constructor, copy and assignment to prevent usage.		
 	protected:
 		//! Do not use
-		MemoryManager<T>() {}
+		MemoryManager() {}
```

Comment:
```suggestion
		MemoryManager() = delete;
```
Since the intent seems to be to prohibit usage